### ISSUE TITLE
logging.Logger missing "filters"

### ISSUE THREAD

**Issue by shish** [python/typeshed#1539]
```
$ cat test.py
import logging
print(logging.getLogger().filters)

$ python test.py
[]

$ mypy test.py
test.py:2: error: "Logger" has no attribute "filters"; maybe "filter"?
```

https://github.com/python/typeshed/blob/10a3e43a192c91d088f7a9607afa7e07408e7e02/stdlib/2and3/logging/__init__.pyi#L44

**Comment by OddBloke** [comment]
This looks like it was fixed in master in #1448.

**Comment by shish** [comment]
Aha, I shall try to figure out how to update my local copy to bring it in sync, thanks :)
